src/hmpb/findTargets.ts

```diff
@@ -36,37 +38,66 @@ export default function* findTargets(
       continue
     }
 
-    const actualAspectRatio = shape.bounds.width / shape.bounds.height
-    if (
-      actualAspectRatio < minAspectRatio ||
-      actualAspectRatio > maxAspectRatio
-    ) {
-      debug(
-        'skipping shape because it is the wrong aspect ratio: %d ≉ %d ± %d: %O',
-        actualAspectRatio,
-        aspectRatio,
-        aspectRatioTolerance,
-        shape.bounds
-      )
-    } else if (
-      shape.bounds.width < expectedWidth - errorMargin ||
-      shape.bounds.width > expectedWidth + errorMargin
-    ) {
-      debug(
-        'skipping shape because it is the wrong width: %d ≉ %d ± %d: %O',
-        shape.bounds.width,
-        expectedWidth,
-        errorMargin,
-        shape.bounds
-      )
+    // If we already found one, so let's use it to determine the right size.
```

Comment:
Added this change in order to ensure the regression tests passed with the previous changes. Should make template scanning more reliable in general.